service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -0,0 +1,41 @@
+[id="updating-ossm"]
+= Updating {ProductName} from version 1.0.1 to 1.0.2
+include::modules/ossm-document-attributes.adoc[]
+:context: installing-ossm
+toc::[]
+
+Updating {ProductName} requires extra steps before you update OpenShift to version 4.2. 
```

Comment:
Throughout:

s/OpenShift/{product-title}/

Unless there was a reason to not do this in Service Mesh docs specifically.